src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnBeregnYtelseStegIntegrationTest.kt

```diff
@@ -331,25 +331,6 @@ class TilsynBarnBeregnYtelseStegIntegrationTest(
             assertHarPerioderForJanuarTilMars(revurdering.id)
         }
 
-        /**
-         * Inget reellt tilfelle, men verifiserer at man ikke gjenbruker perioder før revurder-fra-datoet
-         * I en revurdering er det egentlige ikke mulig å legge til perioder før revurder-fra
-         */
-        @Test
-        fun `skal ikke ta med perioder før revurder-fra`() {
-            innvilgPerioderForJanuarOgFebruar(behandling.id)
-            assertHarPerioderForJanuarOgFebruar(behandling.id)
-
-            testoppsettService.oppdater(behandling.copy(status = BehandlingStatus.FERDIGSTILT))
-            val revurdering = opprettRevurdering(revurderFra = april.atDay(15))
-
-            innvilgPerioderForMars(revurdering)
-
-            // Har ikke med periodene for mars som ble lagt inn i revurderingen
-            // Men har kopiert perioder for januar og februar fra forrige behandlingen
-            assertHarPerioderForJanuarOgFebruar(revurdering.id)
-        }
```

Comment:
Tar gjerne tilbakemeldinger på om det er greit å fjerne denne testen. Slik som koden er nå vil ikke tilfellet i testen kunne intreffe.

Reply:
Ja tenker den er god 👍 
Vil tro at man kunde ha endret til å ha periode for jan, feb og april (april er nytt)
Så revurdere fra 15 april, legger inn perioder for mars (som i koden)
Verifisere at mars ikke er med

Men mulig det fortsatt ikke helt gir verdi